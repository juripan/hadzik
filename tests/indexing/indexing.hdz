znak f = 'a'
naj str = "012345\n"
// TODO add reassign to indexes
// str[0] = 's'
sicke(cif i = 0, i < 6, i++){
    znak f = str[i]
    hutor(str)
    hutor(f)
    hutor('\n')
}