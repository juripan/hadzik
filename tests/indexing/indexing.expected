stdout: 012345
0
012345
1
012345
2
012345
3
012345
4
012345
5
| stderr: | returncode: 0